jgiven-testng/src/test/java/com/tngtech/jgiven/testng/concurrency/TestNgInheritedParallelizationTest.java

```diff
@@ -0,0 +1,35 @@
+package com.tngtech.jgiven.testng.concurrency;
+
+import com.tngtech.jgiven.annotation.Pending;
+import com.tngtech.jgiven.testng.ScenarioTest;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+@Test(singleThreaded = false )
+@Ignore("root cause of error not yet fixed")
+public class TestNgInheritedParallelizationTest
+    extends ScenarioTest<Stages.ParallelGivenStage, Stages.ParallelWhenStage, Stages.ParallelThenStage> {
+
+    @Test
+    @Pending
+    public void fistTest() {
+        int i =0; //for (int i = 1; i <= 100; i++) {
```

Comment:
Why leaving the comment here? I think this should at least be done similar to the TestNgInjectedParallelizationTest class.